### ISSUE TITLE
Request: Use artist detected by HPX in chaika search

### ISSUE THREAD

**Issue by NonNov** [happypandax/plugins#44]
I recently encountered a lot of incorrectly tagged libraries which was caused by doujins with really generic names (e.g. "Cheeky") resulting in the plugin picking the wrong chaika gallery for metadata.
Since HPX already recognized and tagged the artist of the doujin correctly during the import (due to the file name), would it be possible to make an optional setting which incorporates the artist tag (if present) in the chaika search?

&nbsp;

_(There hasn't been much activity on this plugin in a while so please let me know if this request is low priority/not worth working on because in that case I'd try to look into this myself.)_

**Comment by twiddli** [comment]
You're in luck. I'm planning on getting back into it.

**Comment by twiddli** [comment]
Did you try using the `filename_search` option on the chaika metadata plugin?

**Comment by NonNov** [comment]
Yes, for example in both cases a file called "[kakao] Cheeky (COMIC Kairakuten BEAST 2019-05).zip" results in the chaika /gallery/39005/ getting fetched, when it should be /gallery/28081/ .

I already (separately to the test above) played around a bit by restricting the search results by adding a "&tags=publisher%3APublisherNameGoesHere" in the title_search entry of the URLS dict, but sadly enough while it did help for some libraries, it didn't for all of them. (The one above for example then fetched /gallery/28862/ instead.) 
And even if it would've worked for me, other people might not be able to add specific restrictions like a publisher filter.
